src/Diffusion.jl

```diff
@@ -1,5 +1,9 @@
 module Diffusion
 
-# Write your package code here.
+export absorb_boundry
+
+function absorb_boundry(pos,r)
+    return pos + r
```

Comment:
This is a great way to see whether you understand git, GitHub, and the testing infrastructure. Obviously this will need a real implementation later!